game/views.py

```diff
@@ -65,23 +67,20 @@ def post(self, *args, **kwargs):
         position = self.request.data.get("position", None)
         player = Player.objects.get(user=self.request.user)
         content_type = ContentType.objects.get(model='player')
-
-        if not position or not position['longitude'] or not position['latitude']:
-            return Response({"message": "Position error"},
-                            status.HTTP_400_BAD_REQUEST)
-
-        content = Content.objects.filter(game=game,
-                                         content_type=content_type,
-                                         content_id=player.pk).first()
-        position_point = Point(position['longitude'], position['latitude'])
-
-        if not content:
-            Content.objects.create(game=game,
-                                   content_type=content_type,
-                                   content_id=player.pk,
-                                   position=position_point)
-            return Response({}, status.HTTP_201_CREATED)
+        data = {
+            'game': game.pk,
+            'content_type': content_type.pk,
+            'content_id': player.pk,
+            'position': position
+        }
+        instance = Content.objects.filter(game=game, content_id=player.pk,
+                                         content_type=content_type)
+        if instance.exists():
+            content_ser = ContentSerializer(instance.first(), data=data)
```

Comment:
Using deserializers is better for create objects in DB because it validates correctly the fields.